package.json

```diff
@@ -79,6 +79,7 @@
     "next": "14.1.1",
     "next-mdx-remote": "^4.4.1",
     "node-fetch": "^3.3.2",
+    "node-fetch-2": "npm:node-fetch@^2.7.0",
```

Comment:
why `npm:node-fetch@^2.7.0` and not just `node-fetch@^2.7.0` ?

also maybe you could also update the other script that uses `node-fetch`, so there will be no need for `node-fetch-2`?


or maybe best solution is just to update https://github.com/asyncapi/website/blob/master/.github/workflows/regenerate-meetings-and-videos.yml#L28 that rungs this script. v4 is using node 20 by default - wouldn't that fix the issue? using v20 instead of v16 of node?

Reply:
The current node-fetch v3 is being used by the website only supports ESM modules, so I try to keep 2 versions.

Another solution is to migrate to ESM from CommonJS, I don't like this idea unless we are using Deno or Webpack to bundle the script, the primary reason is by the end of the day webpack or another tool will always bundle ESM to CommonJS in production.

It reminds me of the old days when this pissed a lot of people including me and almost break the entire javascript ecosystem xD
https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c